packages/jest-jasmine2/package.json

```diff
@@ -14,6 +14,7 @@
     "graceful-fs": "^4.1.11",
     "is-generator-fn": "^1.0.0",
     "jest-diff": "^22.4.0",
+    "jest-each": "^0.5.0",
```

Comment:
To easier roll this in long-lived jest releases, it would be awesome to release 1.0.0 before merging this 🙂